### PR TITLE
Update python example to use PyJWT

### LINKED ISSUE #33796: Documentation for generating a JWT uses relatively unknown Python library
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/.github/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/generating-a-json-web-token-jwt-for-a-github-app#example-using-python-to-generate-a-jwt

### What part(s) of the article would you like to see updated?

The python code relies on the [jwt](https://github.com/GehirnInc/python-jwt) library, which is comparatively unknown.  I think a better choice for github documentation is the more widely-recognized and used [PyJWT](https://github.com/jpadilla/pyjwt) library.  The code would be more simple, and the library is more highly regarded:

|library | github stars | libraries.io SourceRank | snyk Package Health Score |
|----------|-----------------:| :---: | :---: |
| jwt | 137 | 18 | 58 / 100 |
| PyJWT | 4988 | 22 | 91 / 100 |

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by altaurog**
I believe PyJWT library is a better recommendation, judging from ratings on library.io and snyk.  Additionally, the code is slightly more simple.

### Why:

Closes: https://github.com/github/docs/issues/33796

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Python example for generating and signing a JWT

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by Lockaut04** [comment]
Xxx

**Comment by nguyenalex836** [comment]
@altaurog Thanks so much for opening a PR! I'll get this triaged for review ✨

File: content/apps/creating-github-apps/authenticating-with-a-github-app/generating-a-json-web-token-jwt-for-a-github-app.md [hunk 1]
```diff
@@ -84,15 +84,16 @@ puts jwt
 
 {% note %}
 
-**Note:** You must run `pip install jwt` to install the `jwt` package in order to use this script.
+**Note:** You must run `pip install PyJWT` to install the `jwt` package in order to use this script.
```
Comment (m-balghonaim) [inline content/apps/creating-github-apps/authenticating-with-a-github-app/generating-a-json-web-token-jwt-for-a-github-app.md:87]:
```suggestion
**Note:** You must run `pip install PyJWT` to install the `PyJWT` package in order to use this script.
```

Comment (nguyenalex836) [reply content/apps/creating-github-apps/authenticating-with-a-github-app/generating-a-json-web-token-jwt-for-a-github-app.md:87]: @m-balghonaim Thank you for this suggestion! 💛 

@altaurog If you are able to commit the suggestion above, we can go ahead and get this merged ✨ thank you!

Comment (altaurog) [reply content/apps/creating-github-apps/authenticating-with-a-github-app/generating-a-json-web-token-jwt-for-a-github-app.md]: haha, good catch.  Done with thanks.

**Review by nguyenalex836 — APPROVED** [review]

**Pull request merged**